Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+MAINTAINER Shikher Verma "root@shikherverma.com"
+RUN apt-get update -y --fix-missing
+RUN apt-get install -y python python-pip python-dev build-essential git nodejs npm ruby ruby-dev
+# Install Dependencies
+RUN \
+  apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10 && \
```

Comment:
You probably want to use hkps? Does the ubuntu keyserver not support it?

Reply:
Not sure what `hkps` is. Will look into it.